### PR TITLE
Feat: added notes app UI

### PR DESCRIPTION
**Pull request by paarthbajaj**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,51 +1,43 @@
...
-  height: 40vmin;
-  pointer-events: none;
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+@import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
+/* @import url(https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css); */
```
Comment (Nagashree01) [inline src/App.css:2]: you can remove the comment if you are not using later

File: src/App.css [hunk 2]
```diff
@@ -1,51 +1,43 @@
...
-
-.links a {
-  color: #d9d9d9;
-  font-size: 1.2rem;
-  margin: 1rem;
+.App {
+  background: #ebebeb;
+  min-height: 90vh;
 }
+@import url(https://patched-ui.netlify.app/utility-classes.css);
```
Comment (Nagashree01) [inline src/App.css:43]: you can write all  the imports together at the start. Looks clean

**Review by Nagashree01 — APPROVED** [review]
Code is clean and well written.  Good to merge.
You can add a screenshot in description or host it as UI could be seen.

File: src/App.css (hunk 1 above)
Comment (Kothalkarpriya) [reply src/App.css:2]: yes remove the unwanted commented code

File: src/App.js [hunk 3]
```diff
@@ -1,41 +1,21 @@
...
-        </div>
-      </header>
+      <Navbar />
+      <div className="app-container pt-1">
+        <Sidebar />
+        <Searchbar />
+        <NoteCard />
+      </div>
+
+      <Routes>{/* <Route path="/" element={< />} /> */}</Routes>
```
Comment (Kothalkarpriya) [inline src/App.js:18]: comments to be removed if not in use

**Review by Kothalkarpriya — APPROVED** [review]
Your code looks clean and good, just remove the unwanted comments and import at the beginning

File: src/App.js (hunk 3 above)
Comment (paarthbajaj) [reply src/App.js:18]: Will remove them at the end for sure. Thanks for pointing it out.

File: src/App.css (hunk 1 above)
Comment (paarthbajaj) [reply src/App.css:2]: Will remove the unwanted comments later for sure.

**Pull request merged**